$MirOS: ports/books/selfhtml/patches/patch-src_selfnosize_css,v 1.1 2005/03/27 17:26:25 tg Exp $

	He says this file does not contain fixed sizes... well, it did.

--- src/selfnosize.css.orig	Tue Feb 27 18:00:17 2007
+++ src/selfnosize.css	Sat Dec  6 15:51:02 2014
@@ -14,7 +14,7 @@ h2 { margin-bottom:18px; }
 
 li,dt { margin-top:3px; }
 
-pre { font-family:"Courier New",Courier,monospace; font-size:13.4px; color:#0000C0; background-color:#FFFFE0; border:10px solid #EEEEEE; padding:10px; white-space:pre; }
+pre { font-family:"Courier New",Courier,monospace; color:#0000C0; background-color:#FFFFE0; border:10px solid #EEEEEE; padding:10px; white-space:pre; }
 
 td pre {border-style:none; padding:0;}
 
